src/java.base/share/classes/javax/crypto/CipherInputStream.java

```diff
@@ -30,38 +30,40 @@
 import java.io.InputStream;
 
 /**
- * A CipherInputStream is composed of an InputStream and a Cipher so
- * that read() methods return data that are read in from the
- * underlying InputStream but have been additionally processed by the
- * Cipher.  The Cipher must be fully initialized before being used by
- * a CipherInputStream.
+ * A <code>CipherInputStream</code> is composed of an <code>InputStream</code>
```

Comment:
Use `{@code }`? Applies to the rest of changes of this file.

Reply:
using `{@code }`